tests/test_more.py

```diff
@@ -21,15 +20,15 @@
     product,
     repeat,
 )
-from operator import add, mul, itemgetter
-from pickle import loads, dumps
+from operator import add, itemgetter, mul
```

Comment:
No one cares about the components of a from-import being alphabetized.

In some ways, this is worse.  The `add` and `mul` are semantically related.